Nodejs/Product/Nodejs/TypingsAcquisition.cs

```diff
@@ -78,37 +78,49 @@ internal class TypingsAcquisition {
 
         public Task<bool> AcquireTypings(IEnumerable<string> packages, Redirector redirector) {
             return typingsToolGlobalWorkSemaphore.WaitAsync().ContinueWith(async _ => {
-                var typingsToAquire = GetNewTypingsToAcquire(packages);
-                var success = await DownloadTypings(typingsToAquire, redirector);
-                if (success) {
-                    _acquiredTypingsPackageNames.Value.UnionWith(typingsToAquire);
+                try {
+                    return await DownloadTypingsForProject(redirector) && await DownloadTypingsForPackages(packages, redirector);
```

Comment:
Let's flip these two around so that typings.json has the "final say" because if someone has a typings.json file, they've explicitly chosen to lock their dependency versions.